examples/multimodal/rag.ts

```diff
@@ -41,7 +41,7 @@ async function main() {
 
   const queryEngine = index.asQueryEngine({
     responseSynthesizer: new MultiModalResponseSynthesizer(),
-    retriever: index.asRetriever({ similarityTopK: 3, imageSimilarityTopK: 1 }),
+    retriever: index.asRetriever({ topK: { text: 3, image: 1 } }),
```

Comment:
should probably better use uppercase here
```suggestion
    retriever: index.asRetriever({ topK: { TEXT: 3, IMAGE: 1 } }),
```